Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Zygote"
 uuid = "e88e6eb3-aa80-5325-afca-941959d7151f"
-version = "0.5.14"
+version = "0.5.15"
```

Comment:
really we should avoid this, open PRs will continuously have conflicts with master, really annoying

Reply:
🤷 Its just not that big a deal.
it takes 30 seconds to rebase.

The person it annoys is the PR author, and that is me.
And I am not annoyed.
If other people don't want to rebase to deal with it then they don't have to bump the version number in the PR.
I am not here telling anyone else to.

On a small PR like this one the expectation is that it would be merged quickly (and this would have been except I was on leave).
and in a big PR the time to rebase because the version number has changed is nothing compared to the time taken to create the PR.

Further more, rebasing pulls in the changes from master.
And it is possible to end-up with a senario that has two PRs can both pass tests on their own, but not pass tests if merged one after the other.
Rebasing prevents that.
(Some CI (definately Bors) also prevents that by testing on the result of the merge, rather than testing the actual code in the branch.)

Reply:
I agree, just making sure we don't force this habit on contributors